libs/design/src/atoms/button/button.component.scss

```diff
@@ -38,9 +38,9 @@
         }
     }
 
-    &.daff-accent-blue {
-        background-color: $accent-blue;
-        border: 1px solid $accent-blue;
+    &.daff-blue {
```

Comment:
Still no.

Reply:
We didn't talk about this part yesterday. How do we style colors that are not $primary and $accent then? I still want to be able to add a `blue` variant to button, hero, callout.

Reply:
If you want, you can add a "tertiary" color to the palette.